js_modules/dagster-ui/packages/ui-core/src/ui/Filters/__tests__/useTimeRangeFilter.test.tsx

```diff
@@ -93,18 +91,20 @@ describe('CustomTimeRangeFilterDialog', () => {
     const {result} = renderHook(() => useTimeRangeFilter(mockFilterProps));
     let filter = result.current;
 
-    const {getByText} = render(
-      <CustomTimeRangeFilterDialog filter={filter} closeRef={{current: () => {}}} />,
+    const {getByText} = await act(async () =>
```

Comment:
the test fails without `await act` wrapping the render. This is due to rendering in React 18 being broken up